### PR TITLE
(master)JZ-28189: fix issue when selecting content group with special characters

### PR DESCRIPTION
**Pull request by ouhryn**
jira https://coupadev.atlassian.net/browse/JZ-28189

- [ ] @johnny-lai

### CODE REVIEW

File: coffee/lib/data-source.coffee [hunk 1]
```diff
@@ -12,10 +12,10 @@ class DataSource
   get_option_element_by_value: (value) ->
     # Check if option already exists
     e = $(@form_field)
-    option = e.find('option[value="' + this.escape_html(value) + '"]')
+    option = e.find('option[value="' + value + '"]')
```
Comment (ouhryn) [inline coffee/lib/data-source.coffee]: tested few options/values , and `e` always includes not escaped value

File: coffee/lib/data-source.coffee [hunk 2]
```diff
@@ -12,10 +12,10 @@ class DataSource
...
     if !option.length
-      e.append('<option value="' + this.escape_html(value) + '"></option>')
-      option = e.find('option[value="' + this.escape_html(value) + '"]')
+      e.append($("<option />").attr("value", value).text(value));
```
Comment (ouhryn) [inline coffee/lib/data-source.coffee]: we shouldn't escape value, because it will be sent to the server and we won't be able to find it in the db.

**Comment by johnny-lai** [comment]
thank you. lgtm

**Comment by johnny-lai** [comment]
[lockit v4.8.0] merged via Lockit CLI on 2019-01-08 10:15:39.80241 -0800 PST m=+6.077336042

**Pull request merged**